### ISSUE TITLE
Melee attacks are not honoring DamageReductionMultiplierMelee and MeleeDamageMultiplierVehicle statistics

### ISSUE THREAD

**Issue by IceRaptor** [BattletechModders/CBTBehaviorsEnhanced#63]
From rogueticket-731:

@FrostRaptor you said you use melee attack fir first, and imaginary lasers for otgers, does that mean that thsi only works against first attack?
"statName": "DamageReductionMultiplierMelee",
                "operation": "Float_Add",
                "modValue": "-0.2",
                "modType": "System.Single"
and is this working at all?
              "statName": "ToHitThisActor",
                "operation": "Float_Add",
                "modValue": "1.0",
                "modType": "System.Single",
                                "targetCollection": "Weapon",
                "targetWeaponSubType": "Melee"
Harkonnen08/08/2020
also, does CBTBE checks for destroyed actuators and penalise, or ME critical effect is needed?

FrostRaptor08/08/2020
Sorry, got pulled into another discussion
DamageReductionMultiplierMelee is applied on the target, not the attacker, I presume? If so, it's also only applying to the first attack - but that feels like a bug
Not exactly straightforward to fix though; I'd have to apply the 'real' damage to the weapon, then modified damage as the clustered hits. But I could fix that.
As far as actuators, yes - CBTBE does apply damage reduction for destroyed actuators. It's a modifier controlled through mod.json, with each actuator reducing damage by a set amount - as per the CBT rules.
No need for a ME critical effect, I'm applying it directly

@FrostRaptor i was thinking about DamageReductionMultiplierMelee, and i think that if you are going to fix that, it would make sense to actually replace that with CBTBE specifics stats per attack type. As i cleraly can see f.e shields provdiding resistance against charge specificaly, or reimforced legs against kicks....

**Comment by IceRaptor** [comment]
From rogueticket-839:
```
@FrostRaptor its seems taht melee attacks are not honoring     "MeleeDamageMultiplierVehicle": 1.5,

Hurm.... so I assume you mean Thug vs. Tseudo tank
Thug Punch damage is calculated at:
14:17:41.257 PUNCH baseDamage: 0.5 x attacker tonnage: 80 = 40
14:17:41.257  - Left arm actuator damage is: 1
14:17:41.257  - Right arm actuator damage is: 1
14:17:41.257  - Using arm actuator damage reduction of: 1
14:17:41.257  - Target damage per strike => final: 77 = (raw: 40 + mod: 0) x multi: 1.91664 x reductionMulti: 1
(on a side note, Physical Weapon damage is 395!?! holy crap)
Anywho, the punch attack resole in cleaned_output has:

So it does 77 but is modified up to 97.5
HarkonnenToday at 11:05 AM
it was pyhisacl attack. yes thug versus Tetsudo
FrostRaptorToday at 11:05 AM
Which is... 1.266 modifier?
It then fires 4 more AI laser attacks at 65 damage each
HarkonnenToday at 11:10 AM
yeh, the first one is actual melle and is affe cted, but inmaginary lasers iare not
FrostRaptorToday at 11:10 AM
Yeah
HarkonnenToday at 11:10 AM
97.5 is 65x1.5
FrostRaptorToday at 11:10 AM
14:17:21.881 == Resolving cluster damage, instability, and unsteady on melee attacker: Thug THG-10E_Slapdash_EC1A10DE and target: Testudo Siege Tank_Paladin_6158E907.
14:17:21.881  -- AttackSequence: 868 hit the target.
14:17:21.881  -- Applying 260 damage to target as 4 clusters.
14:17:21.882   Attack direction is: FromLeft
14:17:21.882   65 damage to location: Turret
14:17:21.884   65 damage to location: Turret
14:17:21.884   65 damage to location: Turret
14:17:21.885   65 damage to location: Left
14:17:21.885 == Done.
So yeah, that's the exact bug.
```

**Comment by IceRaptor** [comment]
btw, there is another setting against tirrets like this, probably doesnt wotk either
FrostRaptorToday at 11:11 AM
Yup, cluster damage probably isn't honoring any damage multis on target
Including acid, I bet

**Comment by IceRaptor** [comment]
Should be fixed in 0.5.13
